### PR TITLE
Add local Rari pool deployment

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: scripts/deposit.local.fork.ts [hunk 1]
```diff
@@ -36,53 +25,10 @@ class DepositLocalForked {
     console.log('Sent 10000 cDAI to owner address');
     await this.sendTransaction(250000, depositConfig.holders.stETH, this.owner.address, tokenMap.get('stETH'));
     console.log('Sent 250000 stETH to owner address');
-
-    for (let i = 0; i < depositConfig.addresses.tempusPools.length; i++) {
```
Comment (yuval-weiss) [inline scripts/deposit.local.fork.ts:40]: Why was this deleted?

Comment (yuval-weiss) [inline scripts/deposit.local.fork.ts:40]: Why was this deleted?

Comment (lazarmitic) [reply scripts/deposit.local.fork.ts:40]: App now works with pools that do not have any liquidity, we can now provide initial liquidity using the app.

Comment (lazarmitic) [reply scripts/deposit.local.fork.ts:40]: App now works with pools that do not have any liquidity, we can now provide initial liquidity using the app.

Comment (yuval-weiss) [reply scripts/deposit.local.fork.ts:40]: noice

Comment (yuval-weiss) [reply scripts/deposit.local.fork.ts:40]: noice, lol.

**Review by yuval-weiss — APPROVED** [review]
